source/features/latest-tag-button.tsx

```diff
@@ -95,7 +95,9 @@ async function init(): Promise<false | void> {
 	if (pageDetect.isRepoRoot()) {
 		href = `/${getRepoURL()}/tree/${latestTag}`;
 	} else {
-		href = replaceBranch(currentBranch, latestTag);
+		const pathnameParts = parseRoute(location.pathname);
+		pathnameParts.branch = latestTag;
+		href = pathnameParts.toString();
```

Comment:
What about this pattern for assignment? (Only if it's done all in one place; don't refactor other code that has conditions, etc)
```suggestion
		href = {
			...parseRoute(location.pathname),
			branch: latestTag
		}.toString();
```